File: httpsnippet/src/test/resources/output/python/python3/full.py
import http.client

conn = http.client.HTTPConnection("mockbin.com")

payload = "foo=bar"

headers = {"Cookie":"foo=bar; bar=baz","content-type":"application/x-www-form-urlencoded","accept":"application/json"}

conn.request("POST", "/har?baz=abc&foo=bar&foo=baz&key=value", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))

File: httpsnippet/src/test/resources/output/clojure/clj_http/full.clj
(require '[clj-http.client :as client])

(client/post "http://mockbin.com/har?key=value" {headers {Cookie "foo=bar;bar=baz" 
                                                   :content-type "application/x-www-form-urlencoded" 
                                                   :accept "application/json" 
                                                   } 
                                                :form-params {foo "bar" 
                                                    } 
                                                :query-params {foo ["bar" "baz" ] 
                                                     :baz ["abc" ] 
                                                     } 
                                                :accept :json 
                                                })